nf_core/pipeline-template/.github/workflows/branch.yml

```diff
@@ -22,7 +24,7 @@ jobs:
         uses: mshick/add-pr-comment@b8f338c590a895d50bcbfa6c5859251edc8952fc # v2
         with:
           message: |
-            ## This PR is against the `master` branch :x:
+            ## This PR is against the `${{github.event.pull_request.base.ref}}` branch :x:
```

Comment:
wait! sorry I think this won't work, because this action is on `pull_request_target`, not `pull_request` 🙁 I thought we could use the env variable directly

Reply:
> The pull_request_target event provides the same GitHub context variables as the pull_request event.

Reply:
ok let's do it 😄